chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
-to create functionally-named variables,
-and to sensibly subset data for analysis.
+We have a few recommendations on how to use this command for data cleaning.
+First, we suggest keeping the same variable names as in the survey instrument,
+so it's easy to connect the two files.
+Don't skip the labelling.
+Applying labels makes it easier to understand what the data is showing while exploring the data. 
+This minimizes the risk of small errors making their way through into the analysis stage.
+Variable and value labels should be accurate and concise.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning\#Applying\_Labels}}
+Recodes should be used to turn codes for ``Don't know'', ``Refused to answer'', and
+other non-responses into extended missing values.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning\#Survey\_Codes\_and\_Missing\_Values}}
+String variables need to be encoded, and open-ended responses, categorized or dropped\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning\#Strings}}
+(unless you are using qualitative or classification analyses, which are less common).
+Finally, any additional information collected only for quality monitoring purposes,
+such as notes and duration fields, can also be dropped.
+
+% Outputs -----------------------------------------------------------------
+
+% Data set
+The most important output of data cleaning is the cleaned data set. 
+It should contain the same information as the raw data set,
+with no changes to data points.
+It should also be easily traced back to the survey instrument,
+and be accompanied by a dictionary or codebook.
+Typically, one cleaned data set will be created for each data source,
+i.e. per survey instrument.
+Each row in the cleaned data set represents one survey entry or unit of observation.\sidenote{\cite{tidy-data}}
+If the raw data set is very large, or the survey instrument is very complex,
+you may want to break the data cleaning into sub-steps, 
+and create intermediate cleaned data sets
+(for example, one per survey module).
+Breaking cleaned data sets into the smallest unit of observation inside a roster
+make the cleaning faster and the data easier to handle during construction.
+But having a single cleaned data set will help you with sharing and publishing the data.
+To make sure this file doesn't get too big to be handled,
+use commands such as \texttt{compress} in Stata to make sure the data
+is always stored in the most efficient format.
+
+% Documentation
+Throughout the data cleaning process, you will need inputs from the field, 
+including enumerator manuals, survey instruments, 
+supervisor notes, and data quality monitoring reports.
+These materials are part of what we call data documentation
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Documentation}}
+\index{Documentation},
+and should be stored in the corresponding folder, 
+as you will probably need them during analysis and publication.
```

Comment:
We should take a stronger position on this. These materials are essential for data documentation, and must be made available for publication.

Reply:
changed the wording in b9cc5f1